src/app/shared/components/filters-list/user-radius-set/user-radius-set.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, OnInit } from '@angular/core';
+import { ChangeContext } from '@angular-slider/ngx-slider';
+import { FormControl } from '@angular/forms';
+import { Store } from '@ngxs/store';
+import {
+  GetFilteredWorkshops,
+  SetRadiusSize
+} from '../../../store/filter.actions';
+import { DestroyableDirective } from '../../../directives/destroyable.directive';
+import { debounceTime, takeUntil } from 'rxjs/operators';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Component({
+  selector: 'app-user-radius-set',
+  templateUrl: './user-radius-set.component.html',
+  styleUrls: ['./user-radius-set.component.scss']
+})
+export class UserRadiusSetComponent
+  extends DestroyableDirective
+  implements OnInit
+{
+  public defaultValue: number = 5;
+  public options = { floor: 2, ceil: 10 };
+  public currentRadius: FormControl = new FormControl({
+    value: '',
+    disabled: true
+  });
+  8;
```

Comment:
?

Reply:
As I understand requirements, in the input we should only watch a current number of kilometers, but not change it.

Reply:
I am taking about line 28